drivers/net/loopback.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2015 Intel Corporation
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * Network loopback interface implementation.
+ */
+
+#define SYS_LOG_DOMAIN "netlo"
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_NETLO_LEVEL
+#include <logging/sys_log.h>
+
+#include <misc/printk.h>
+
+#include <net/net_pkt.h>
+#include <net/buf.h>
+#include <net/net_ip.h>
+#include <net/net_if.h>
+
+int loopback_dev_init(struct device *dev)
+{
+	return 0;
+}
+
+static void loopback_init(struct net_if *iface)
+{
+	net_if_set_link_addr(iface, "123456", 6, NET_LINK_DUMMY);
```

Comment:
Although the mac address value does not really matter here, we could use the addresses specified for documentation. The slip driver is using these numbers so we could do things similarly.
See RFC 7042 chapter 2.1.1 https://tools.ietf.org/html/rfc7042#section-2.1.1 for details.

Reply:
So, Linux `lo` interface doesn't have any "MAC addresses". I make this call, because that's the call which puts the iface "up". And I initially tried to pass 1-byte MAC address, but anything less than 6 bytes leads to a crash in some thread later.

I'll switch to an address you refer to, but my idea was to use something simple and static.

Reply:
We could leave it like this if needed, I was just pondering this thing aloud.

Reply:
Switched to 00-00-5E-00-53-FF.